src/SprykerSdk/Acp/Manifest/Builder/AppManifestBuilder.php

```diff
@@ -61,14 +61,118 @@ protected function getManifest(ManifestRequestTransfer $manifestRequestTransfer)
         return [
             'name' => $manifestTransfer->getNameOrFail(),
             'provider' => $manifestTransfer->getNameOrFail(),
-            'description' => '',
-            'descriptionShort' => '',
-            'configureUrl' => '',
-            'categories' => [],
-            'assets' => [],
-            'resources' => [],
-            'pages' => [],
-            'label' => [],
+            'description' => 'A short description to be shown below the app name with a promotional text',
```

Comment:
Can't you use the example file and merge in what we get from the console command?